### PR TITLE
Enable feature Utility

### PR DESCRIPTION
**Pull request by kasunsjc**

### CODE REVIEW

File: feature4/feature4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is feature 4 begin");
```
Comment (kasunsjcdemo) [inline feature4/feature4.txt:1]: change hear

**Review by kasunsjcdemo — CHANGES_REQUESTED** [review]
change the content

**Comment by kasunsjc** [comment]
@kasunsjcdemo Done

**Review by kasunsjc — COMMENTED** [review]
done

**Review by kasunsjcdemo — COMMENTED** [review]
done

**Pull request merged**